worker/src/main/java/gov/cms/ab2d/worker/properties/PropertiesInit.java

```diff
@@ -12,9 +12,18 @@
 @Slf4j
 public class PropertiesInit {
 
-    // Load all of the properties from the database
+    private ConfigurableEnvironment configurableEnvironment;
+
+    private PropertiesService propertiesService;
+
     public PropertiesInit(PropertiesService propertiesService, ConfigurableEnvironment configurableEnvironment) {
-        log.info("Adding properties");
+        this.propertiesService = propertiesService;
+        this.configurableEnvironment = configurableEnvironment;
+    }
+
+    // Load all of the properties from the database and insert/overwrite
+    public void updatePropertiesFromDatabase() {
+        log.info("Updating properties");
```

Comment:
Let's change to `debug` or `trace` to avoid excessive logging in the environments.